app/controllers/surveys_controller.rb

```diff
@@ -0,0 +1,6 @@
+# find chosen survey and redirect to its questions
+get 'survey/:id' do
+  survey = TakenSurvey.new(survey_id: params[:id], user_id: current_user.id)
```

Comment:
thank you for the review, according to the helper method, current_user is just set to User.find so it does need id to be called on it